nvautoinstall.spec

```diff
@@ -7,13 +7,17 @@ Summary: NVIDIA Auto Installer for Fedora
 
 License: GPLv3
 Url: https://github.com/t0xic0der/%{srcname}
-Source0: https://github.com/t0xic0der/%{srcname}/releases/download/%{version}/%{name}-%{version}.tar.gz
+Source0: https://github.com/t0xic0der/%{srcname}/releases/download/v%{version}/%{name}-%{version}.tar.gz
```

Comment:
Using the following replacements,

1. `srcname` - `nvidia-auto-installer-for-fedora`
2. `name` - `nvautoinstall`
3. `version` - `0.3.8`

`https://github.com/t0xic0der/nvidia-auto-installer-for-fedora/releases/download/0.3.8/nvautoinstall-0.3.8.tar.gz` throws a `404 Not Found` while `https://github.com/t0xic0der/nvidia-auto-installer-for-fedora/releases/download/v0.3.8/nvautoinstall-0.3.8.tar.gz` gets the tarball just fine.

Thanks for the change.

Reply:
:+1: